cicd/gitops/monitoring-argo-cd-instances.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="monitoring-argo-cd-instances"]
+= Monitoring Argo CD instances
+include::_attributes/common-attributes.adoc[]
+:context: monitoring-argo-cd-instances
+
+toc::[]
+
+By default, the {gitops-title} Operator automatically detects an installed Argo CD instance in your defined namespace, for example, `openshift-gitops`, and connects it to the monitoring stack of the cluster to provide alerts for out-of-sync applications.
+
+.Prerequisites
```

Comment:
@masauso-lungu Typically, the prerequisites belong in the module with the procedure. With the prereqs outside if the module, if the user gets to the module using search, the user might miss the prereqs. I see that putting the prereqs at the top of the module, in the assembly, seems to be common in the Argo docs. If you feel this presentation is correct, please let me know.

Reply:
@mburke5678 Yes, this is common. As per the https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs, if a prerequisite applies to all of the procedures in a user story, we can list the prerequisite in the assembly file. Thanks!